src/main/features/claim/routes/total.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express'
+import { Paths } from 'claim/paths'
+import FeesClient from 'fees/feesClient'
+import { Amount } from 'forms/models/amount'
+import ErrorHandling from 'common/errorHandling'
+
+function renderView (res: express.Response, feeAmount: number, claimAmount: Amount): void {
+  res.render(Paths.claimTotalPage.associatedView, {
+    feeAmount: feeAmount,
+    claimAmount: claimAmount
+  })
+}
+
+export default express.Router()
+
+  .get(Paths.claimTotalPage.uri,
```

Comment:
it's cleaner, damian have changed  most of their this kind of check to async and await